# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471185050 -3600
#      Sun Aug 14 15:30:50 2016 +0100
# Node ID aed778a98a9b66ab7759a978eb2d930fed8c6c84
# Parent  8bceceff4152995846d61539c238f5f433c1d723
Win32: Fix incorrect type of `buf` parameter to EmbeddedR_ReadConsole()

It is `unsigned` but should not be.

diff --git a/rpy/rinterface/_rinterface.c b/rpy/rinterface/_rinterface.c
--- a/rpy/rinterface/_rinterface.c
+++ b/rpy/rinterface/_rinterface.c
@@ -491,7 +491,7 @@
              " (see set_readconsole)");
 
 static int
-EmbeddedR_ReadConsole(const char *prompt, unsigned char *buf, 
+EmbeddedR_ReadConsole(const char *prompt, char *buf,
                       int len, int addtohistory)
 {
   PyObject *arglist;
